### DISCUSSION TITLE
Resize problem

### DISCUSSION THREAD

**Discussion by bero** [gravitational/teleport#40075] [Q&A] [answered]
I am a member in a team that have Windows in GCP (Google Cloud Platform) and I use Teleport.
I use Ubunt0 23.10 as OS and KVM 8.0.4 as virtualization.
GCP is Windows Server 2022 Datacenter version 21H2.

Generally it works well but there are some problems. To get discussion more focused I only write one problem in each thread.

When hover over windows border pointer don't change shape. This can be frustrating as I am used to how resize window normally works. Now I must position pointer exactly over the border to get resize working.

**Comment by zmb3** [comment, marked as answer]
This feature was released in 
15.1.
